common/web/lm-worker/package.json

```diff
@@ -1,7 +1,18 @@
 {
   "name": "@keymanapp/lm-worker",
   "description": "Keyman/Predictive Text worker",
-  "main": "build/index.js",
+  "main": "./build/lib/worker-main.wrapped.js",
+  "exports": {
+    "./worker-main.wrapped.js": "./build/lib/worker-main.wrapped.js",
+    "./build/lib/worker-main.wrapped.js": "./build/lib/worker-main.wrapped.js",
+    "./worker-main.wrapped.min.js": "./build/lib/worker-main.wrapped.min.js",
+    "./build/lib/worker-main.wrapped.min.js": "./build/lib/worker-main.wrapped.min.js"
```

Comment:
Same as with the other comment - our current version of TypeScript does not know how to read from the export mapping, so we need an unaliased export version for now to use in the source and when compiling.